release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2267,6 +2267,8 @@ The `machine-config-daemon` applies the performance profile configuration to the
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021151[*BZ#2021151*])
 
+* It is not possible to create a macvlan on the physical function (PF) when a virtual function (VF) already exists. This issue affects the Intel E810 NIC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2120585[BZ#2120585])
```

Comment:
MacVLan or macvlan?

Reply:
@JoeAldinger  I did a search on docs before adding to see what it is, and it seems to be macvlan across the docs. I think we are OK with macvlan as I think it well understood this is the CNI plugin original content already merged to 4.12 RN so would like to keep this the same.